### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by michalmaj**

### CODE REVIEW

File: python/segmentation/models/model_transformer.py [hunk 1]
```diff
@@ -154,10 +152,10 @@ def __init__(self, in_chans, embed_dims, num_heads, depths,
                 blocks.append(transformer_block(dim=embed_dims[stage_i],
                                                 num_heads=num_heads[stage_i], dropout_p=dropout_p,
                                                 drop_path_p=drop_path_p * (sum(depths[:stage_i]) + i) / (
-                                                            sum(depths) - 1),
```
Comment (caffik) [inline python/segmentation/models/model_transformer.py]: Consider adding a binary operator (i.e. $/$) at the beginning of the new line. 
Reference: https://peps.python.org/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

File: python/segmentation/utils.py [hunk 2]
```diff
@@ -74,28 +65,27 @@ def __init__(self,im, figsize=(12,6), line_color=(255,0,0)):
...
-    def disconnect_mpl(self,_):
+    def disconnect_mpl(self, _):
         self.fig.canvas.mpl_disconnect(self.ka)
 
-
-    def get_bbox_indices(self, scale_factor = None):
+    def get_bbox_indices(self, scale_factor=None):
         pts = np.array(self.selected_points)
-        if(scale_factor is not None):
+        if (scale_factor is not None):
```
Comment (caffik) [inline python/segmentation/utils.py]: Please remove the parentheses

**Review by caffik — COMMENTED** [review]
Some small changes required.

File: python/segmentation/models/model_transformer.py (hunk 1 above)
Comment (michalmaj) [reply python/segmentation/models/model_transformer.py]: I have implemented the changes as suggested in our conversation. Thank you for your input!

File: python/segmentation/utils.py (hunk 2 above)
Comment (michalmaj) [reply python/segmentation/utils.py]: I have implemented the changes as suggested in our conversation. Thank you for your input!

**Review by caffik — APPROVED** [review]
Request accepted.

**Comment by ziobron** [comment]
Your PR was merged!
🏅 2 XP granted. Thanks @michalmaj for making course materials better!

**Pull request merged**